src/Annotator.scss

```diff
@@ -172,6 +161,23 @@
         width: 282px; // Hard-coded width to solve layout issues
     }
 
+    .ba-annotation-list {
+        width: 100%;
+
+        > li {
```

Comment:
These `li` tags have a classname, which can be used directly without nesting to decrease specificity. You could also consider using sibling selectors and list padding to avoid the pseudo-classes:

```
.ba-annotation-list {
    padding-bottom: 10px;
}

.ba-annotation-list-item + .ba-annotation-list-item { 
    border-bottom: 1px solid $seesee; 
}
```